### PR TITLE
feat: send reminder comment to stale initial PRs

### PR DESCRIPTION
**Pull request by rmehner**
Turns out some of our users are confused why Greenkeeper is not enabled
for their repos and it turns out, it's because they never merged the
initial PR. With this, we send out a reminder comment on the PR so
people have a clearer understanding of what happens.

We don't nag them again once we reminded them for that repo. We also
don't comment on very old PRs, as this likely causes more problems for
our users then it does good (super outdated dependencies etc.)

Hope this helps a couple of users to run
Greenkeeper :)

### CODE REVIEW

**Comment by rmehner** [comment]
Somehow @finnp's commit got lost during rebase :/

Another note: I'm no fan of the `await waitFor(500)` in the tests, but without that, `tap` runs `beforeEach` of the next test before `afterEach` of the previous one has been completed, leading to document conflicts. I'm sure there's a better way to handle this and I hope we find this way :)

**Comment by rmehner** [comment]
@christophwitzko also the `schedule-stale-initial-pr-reminders` needs to be scheduled with cron or something, couldn't find any reference how to do that within your infrastructure, any pointers? :)

**Comment by christophwitzko** [comment]
@rmehner Hmm, I think the best way is to use `setTimeout`. Or are there any arguments against it?

File: index.js [hunk 1]
```diff
@@ -63,6 +63,15 @@ require('./lib/rollbar')
...
   }
 
+  const scheduleRemindersJobData = Buffer.from(JSON.stringify({name: 'schedule-stale-initial-pr-reminders'}))
+  setInterval(async () => {
+    try {
+      await scheduleJob(scheduleRemindersJobData, {priority: 1})
+    } catch (e) {
+      console.log(e)
+    }
+  }, 60 * 60 * 1000)
```
Comment (christophwitzko) [inline index.js]: @rmehner set the interval to 1h, is that ok?

Comment (rmehner) [inline index.js]: Guess daily would also be fine

Comment (christophwitzko) [inline index.js]: 👍

File: index.js [hunk 2]
```diff
@@ -63,6 +63,18 @@ require('./lib/rollbar')
     }, 5000)
   }
 
+  const scheduleRemindersJobData = Buffer.from(JSON.stringify({name: 'schedule-stale-initial-pr-reminders'}))
+  async function scheduleReminders () {
+    try {
+      await scheduleJob(scheduleRemindersJobData, {priority: 1})
+    } catch (e) {
+      console.log(e)
```
Comment (Realtin) [inline index.js]: is console logging the error intentional? or should we just throw it?

**Comment by rmehner** [comment]
> @rmehner Hmm, I think the best way is to use setTimeout. Or are there any arguments against it?

Sorry, didn't see that comment earlier. The job itself is as idempotent as possible (with remote calls involved, you can't get that 100% safety there), so it shouldn't matter if it's run twice a day or something. I usually go with what the system offers, as cron is a pretty well-used tool, but I have no qualms about using setTimeout for this. It's also no super critical feature anyway, just something that hopes to help some stuck users.

File: index.js (hunk 2 above)
Comment (christophwitzko) [inline index.js]: if we could use rollbar here, that would be awesome. ☺️

**Pull request merged**